hdfs: formatzk fails when deploying the collection only

Formatzk fails when deploying with tdp-collection without `tdp-lib`.

stderr:
```
Exception in thread "main" java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at sun.instrument.InstrumentationImpl.loadClassAndStartAgent(InstrumentationImpl.java:386)
	at sun.instrument.InstrumentationImpl.loadClassAndCallPremain(InstrumentationImpl.java:401)
Caused by: java.io.FileNotFoundException: /etc/jmx-exporter/zkfc.yml (No such file or directory)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(FileInputStream.java:195)
	at java.io.FileInputStream.<init>(FileInputStream.java:138)
	at java.io.FileReader.<init>(FileReader.java:72)
	at io.prometheus.jmx.shaded.io.prometheus.jmx.JmxCollector.<init>(JmxCollector.java:78)
	at io.prometheus.jmx.shaded.io.prometheus.jmx.JavaAgent.premain(JavaAgent.java:29)
	... 6 more
/bin/sh: line 1: 19953 Done                    echo -e "N"
     19954 Aborted                 | /opt/tdp/hadoop/bin/hdfs --config /etc/hadoop/conf.nn zkfc -formatZK
```

stdout:
```
FATAL ERROR in native method: processing of -javaagent failed
```


jmx exporter playbooks are in the wrong order

--- a/playbooks/hdfs.yml
+++ b/playbooks/hdfs.yml
@@ -14,10 +14,10 @@
 - import_playbook: hdfs_namenode_config.yml
 - import_playbook: hdfs_client_config.yml
 - import_playbook: hdfs_ranger_config.yml
+- import_playbook: hdfs_jmx-exporter_config.yml
 
 - import_playbook: hdfs_ranger_init.yml
 - import_playbook: hdfs_namenode_formatzk.yml
-- import_playbook: hdfs_jmx-exporter_config.yml
 - import_playbook: hdfs_journalnode_start.yml
 - import_playbook: hdfs_namenode_start.yml
 - import_playbook: hdfs_datanode_start.yml

